integration-tests/src/test/resources/sitconfig/java/SitConfigTests.java

```diff
@@ -0,0 +1,554 @@
+        : "Didn't get the expected value "
+            + testConnectionsonReserve
+            + " for testConnectionsonReserve";
+    println("inactiveConnectionTimeoutSeconds:" + jcpb.getInactiveConnectionTimeoutSeconds());
+    assert inactiveConnectionTimeoutSeconds == jcpb.getInactiveConnectionTimeoutSeconds()
+        : "Didn't get the expected value "
+            + inactiveConnectionTimeoutSeconds
+            + " for inactiveConnectionTimeoutSeconds";
+    // testMaxPoolSize(resourceName, jcpb.getMaxCapacity());
+
+    // Assert the jdbc driver param properties
+    JDBCDriverParamsBean jdbcDriverParams = jdbcDataSourceBean.getJDBCDriverParams();
+    println("Data Source URL:" + jdbcDriverParams.getUrl());
+    assert dsUrl.equals(jdbcDriverParams.getUrl())
+        : "Didn't get the expected url for datasource " + dsUrl;
+
+    // println("jdbcDriverParams.getPasswordEncrypted()" +
+    // jdbcDriverParams.getPasswordEncrypted()[0]);
+    // Assert datasource user property
+    /*
+    JDBCPropertiesBean properties = jdbcDriverParams.getProperties();
+    JDBCPropertyBean[] properties1 = properties.getProperties();
+    for (JDBCPropertyBean jDBCPropertyBean : properties1) {
+        println(jDBCPropertyBean.getName());
+    }
+     */
+    // Assert datasource is working with overiridden value
+    DataSource dataSource = getDataSource("jdbc/" + jdbcResourceName);
+
+    try {
+      Connection connection = dataSource.getConnection();
+      Statement stmt = connection.createStatement();
+      ResultSet rs = stmt.executeQuery("SELECT * FROM v$instance");
+      while (rs.next()) {
+        println("INSTANCE_NAME:" + rs.getString("INSTANCE_NAME"));
+        assert dbInstanceName.equals(rs.getString("INSTANCE_NAME"));
+        println("HOST_NAME:" + rs.getString("HOST_NAME"));
+        assert dbHostName.equals(rs.getString("HOST_NAME"));
+      }
+      /*
+      ResultSetMetaData meta = rs.getMetaData();
+      int colCount = meta.getColumnCount();
+      println(colCount + "");
+      for (int col = 1; col <= colCount; col++) {
+          System.out.print(meta.getColumnName(col) + " ");
+      }
+      println("");
+      while (rs.next()) {
+          println("INSTANCE_NAME:" + rs.getString("INSTANCE_NAME"));
+          println("HOST_NAME:" + rs.getString("HOST_NAME"));
+          for (int col = 1; col <= colCount; col++) {
+              Object value = rs.getObject(col);
+              if (value != null) {
+                  System.out.print("@" + value.toString() + "@ ");
+              }
+          }
+          println("");
+      }
+       */
+    } catch (SQLException ex) {
+      Logger.getLogger(SitConfigTests.class.getName()).log(Level.SEVERE, null, ex);
+    }
+  }
+
+  public void testSystemResourcesJDBCAttributeChangeSecret(String jdbcResourceName)
+      throws ManagementException {
+    String dsUrl = "jdbc:oracle:thin:@//slc11smq.us.oracle.com:1583/w18ys12c.us.oracle.com";
```

Comment:
Remove internal URL

Reply:
will replace it with mysql

Reply:
SECURITY VIOLATION
do not check oracle internal server names into public repositories

Reply:
Fixed.